### ISSUE TITLE
keep original width

### ISSUE THREAD

**Issue by unhammer** [lettier/gifcurry#82]
The default setting resized the gif. I don't see a button to let it be the original width or just not change the width. I would expect things not to get changed unless I've actively clicked anything. I don't know what the best UI would be, but perhaps leave numbers at the original values on opening files?

**Comment by pflovv** [comment]
Yes, keeping original size by default is important. Also it would be great if leaving us a way to save the preferences, such as fps, start/end frame, etc.
